core/audits/network-requests.js

```diff
@@ -41,8 +46,11 @@ class NetworkRequests extends Audit {
     // for the general solution to this.
     /** @type {string|undefined} */
     let mainFrameId;
-    if (artifacts.GatherContext.gatherMode === 'navigation') {
-      const mainResource = await MainResource.request({devtoolsLog, URL: artifacts.URL}, context);
+    if (artifacts.GatherContext.gatherMode === "navigation") {
+      const mainResource = await MainResource.request(
+        { devtoolsLog, URL: artifacts.URL },
```

Comment:
this and the change above look like they may be default prettier formatting or something?

Reply:
yep. it was. do y'all have formatter settings that can be applied to VSCode?